### PR TITLE
Adding Errata docs for Katello 2.1

### PR DESCRIPTION
**Pull request by waldenraines**

### CODE REVIEW

File: docs/user_guide/errata/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+To view the Affected Content Hosts of an Errata:
+
+- navigate to: Content > Errata
+- Click on an Errata ID
+- Click on the Content Hosts Tab
+
+Note the following option:
+
+- Checking the box limits the display of Content Hosts to those which already have the Errata available in their Lifecycle Environment.
```
Comment (jlsherrill) [inline docs/user_guide/errata/index.md]: and Content View.

**Comment by waldenraines** [comment]
@jlsherrill updated.

File: docs/user_guide/errata/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+layout: documentation
+title: Errata
+sidebar: sidebars/documentation.html
+---
+
+# Errata
+
+Errata are incremental updates between major releases.  Errata is metadata about a group of packages that explains the importance of the package updates.
```
Comment (jlsherrill) [inline docs/user_guide/errata/index.md]: An Erratum is metadata...

**Comment by jlsherrill** [comment]
ACK pending last comment

**Pull request merged**